### ISSUE TITLE
Core dump during assembly

### ISSUE THREAD

**Issue by apcamargo** [soedinglab/plass#18]
My processes are constantly crashing in the middle of the assembly. I get a `*** glibc detected *** plass: invalid fastbin entry (free): 0x0000000000815a60 ***`.

[plass.segmentation.log](https://github.com/soedinglab/plass/files/3653119/plass.segmentation.log)

I installed Plass through Conda, due to the old libraries of the server I'm working in.

I'm using Plass 2.c7e35, 2TB of memory, 16 × Intel(R) Xeon(R) CPU E5-4650 v2 @ 2.40GHz, Red Hat Enterprise Linux Server release 6.5.

**Comment by AnnSeidel** [comment]
Hi,

**Comment by AnnSeidel** [comment]
Hi,

I fixed some smaller bugs in plass also related to memory access in the last months. Could you try to use the latest version from github?

@martin-steinegger Can you update the conda version of plass? (It's almost one year old)

**Comment by martin-steinegger** [comment]
@AnnSeidel yes its really about time. Should we get the header change in before we make the conda package?
